invisible_cities/dataflow/dataflow.py

```diff
@@ -239,6 +239,19 @@ def spy_count():
     return FutureSpy(future = pair.future, spy = branch(pair.sink))
 
 
+def collect_into_list(arg=None):
+    def _append(l,e):
+        l.append(e)
+        return l
+    pair = reduce(_append, initial=[])()
+    if arg is None:
+        return FutureSpy(future = pair.future, spy = branch(pair.sink))
+    elif _exactly_one(arg):
+        return FutureSpy(future = pair.future, spy = branch(arg, pair.sink))
+    else:
+        raise ValueError("dataflow.collect_into_list only accepts one argument")
+
+
```

Comment:
Is this a leftover from the discussion or are we keeping it for some reason?

Reply:
Same for the tests affecting this

Reply:
Yes, I have yet to push the commits removing the code from `dataflow`.

Reply:
On a side note, since this PR will need some history cleaning, @gonzaponte  do you want to keep these changes in the history if they turn out to be useful in the future dataflow refinements or you prefer to remove them?

Reply:
Good question... We can maybe leave them there, if that doesn't add too much noise.